vscode/src/ruby/chruby.ts

```diff
@@ -179,14 +179,17 @@ export class Chruby extends VersionManager {
       "end",
       `newer_gem_home = File.join(File.dirname(user_dir), "${rubyVersion.version}")`,
       "gems = (Dir.exist?(newer_gem_home) ? newer_gem_home : user_dir)",
-      "data = { defaultGems: Gem.default_dir, gemHome: gems, yjit: !!defined?(RubyVM::YJIT), version: RUBY_VERSION }",
-      "STDERR.print(JSON.dump(data))",
+      `STDERR.print([Gem.default_dir, gems, !!defined?(RubyVM::YJIT), RUBY_VERSION].join("ACTIVATION_SEPARATOR"))`,
     ].join(";");
 
     const result = await this.runScript(
-      `${rubyExecutableUri.fsPath} -W0 -rjson -e '${script}'`,
+      `${rubyExecutableUri.fsPath} -W0 -e '${script}'`,
     );
 
-    return this.parseWithErrorHandling(result.stderr);
+    const [defaultGems, gemHome, yjit, version] = result.stderr.split(
+      "ACTIVATION_SEPARATOR",
```

Comment:
nit: can this value be stored in a variable/constant so it always stay in sync? I know it's unlikely to be changed but I'm just paranoid about it somehow got out of sync and creates mysterious bugs 😅